deltachat-ffi/src/lib.rs

```diff
@@ -1865,23 +1874,30 @@ pub unsafe extern "C" fn dc_chat_is_sending_locations(chat: *mut dc_chat_t) -> l
 
 // dc_msg_t
 
+pub struct MessageWrapper {
```

Comment:
Maybe add some documentation on top describing why `MessageWrapper` is there, such as to prevent usage of the message in the wrong context, or to keep compatibility with the C core.